ansible/cleanup/roles/version/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
I think this role is unnecessary here as the ansible.cfg allows roles from any of the directories cfme-performance has playbooks in.

https://github.com/akrzos/cfme-performance/blob/master/ansible/ansible.cfg#L4